kubernetes-client/test/src/com/goyeau/kubernetes/client/api/ContextProvider.scala

```diff
@@ -1,22 +1,12 @@
 package com.goyeau.kubernetes.client.api
 
 import cats.Parallel
-import cats.effect.std.Dispatcher
 import cats.effect.unsafe.implicits.global
-import cats.effect.{IO, Resource}
-
+import cats.effect.IO
 import scala.concurrent.duration.DurationInt
 
 trait ContextProvider {
-
-  private val dispatcher: Resource[IO, Dispatcher[IO]] = Dispatcher[IO]
-  private val runTimeout                               = 90.seconds
```

Comment:
I am afraid that without timeout, test may run for a loooong time before they actually fail. This may happen when we break some logic which runs asynchronously.